docs/maintenance/database.md

```diff
@@ -42,7 +42,7 @@ PGPASSWORD=$DB_PASSWORD pg_dump -h $SRC_HOSTNAME -p $SRC_HOSTPORT -U $DB_USER --
 
 This will dump the file with the `.dump` extension into the `/tmp` folder.
 
-[For more information and additional options, please check the official documentation.](https://www.postgresql.org/docs/10/app-pgdump.html)
+[For more information and additional options, please check the official documentation.](https://www.postgresql.org/docs/11/app-pgdump.html)<!-- TODO DOCS-612 Confirm link should be updated to the recommended version -->
```

Comment:
Should the database documentation version here match the [recommended version](https://github.com/codacy/chart/pull/798/files#diff-71c6b87612afd48ee2b295f885d377d87f291385b3b5ff796e71947739393994R102)?

Reply:
I updated to version 12 documentation, as it's the recommended version.